Player/Player.gd

```diff
@@ -109,6 +110,12 @@ func _fly(delta: float) -> void:
 	if Input.is_action_pressed("move_left"):
 		direction -= aiming[0]
 	
+	if Input.is_action_pressed("slowmo"):
+		if(slowMo):
+			Engine.time_scale *= 2
```

Comment:
This seems to effect everything except for sound effects. We might want to have a custom audio player class that updates playback speed according to time scale.